app/controllers/v1/supplemental_claims_controller.rb

```diff
@@ -98,11 +100,13 @@ def process_submission
 
           ::Rails.logger.info(post_create_log_msg(appeal_submission_id:, submitted_appeal_uuid:))
           if form4142.present?
-            handle_4142(request_body: req_body_obj,
-                        form4142:,
-                        appeal_submission_id:, submitted_appeal_uuid:)
+            handle_4142(request_body: req_body_obj, form4142:, appeal_submission_id:, submitted_appeal_uuid:)
           end
           submit_evidence(sc_evidence, appeal_submission_id, submitted_appeal_uuid) if sc_evidence.present?
+
+          store_saved_claim(claim_class: SavedClaim::SupplementalClaim, form: req_body_obj.to_json,
```

Comment:
are we losing any data if we don't include `form4142` since that gets deleted from `req_body_obj` on line 92?

Reply:
That's ag good point - I'll save the request with the `form4142` value